gateway/analytics.go

```diff
@@ -336,3 +199,42 @@ func (r *RedisAnalyticsHandler) recordWorker() {
 func DurationToMillisecond(d time.Duration) float64 {
 	return float64(d) / 1e6
 }
+
+func GetGeo(a *analytics.Record, ipStr string, gw *Gateway) {
+	// Not great, tightly coupled
+	if gw.Analytics.GeoIPDB == nil {
+		return
+	}
+
+	record, err := geoIPLookup(ipStr, gw)
+	if err != nil {
+		log.Error("GeoIP Failure (not recorded): ", err)
+		return
+	}
```

Comment:
Can you please skip line after `if` and `for` blocks, it makes the code unreadable.